ras_party/controllers/business_controller.py

```diff
@@ -83,3 +83,23 @@ def businesses_post(business_data, session):
     b = Business.from_party_dict(party_data)
     session.merge(b)
     return b.to_business_dict()
+
+
+@with_db_session
+def businesses_sample_ce_link(sample, link_data, session):
+    """
+    Update business versions to have the correct collection exercise and sample link.
+
+    :param sample: the sample summary id to update.
+    :param link_data: dictionary containing the collectionExerciseId to link with sample.
+    :param session: database session.
+    """
+
+    v = Validator(Exists('collectionExerciseId'))
```

Comment:
Should we do this validation in the controller? I feel like this is something we should do before passing it from the view. This would remove the need to have the default value for payload in the view

Reply:
If link data only has collectionExerciseId in it then we should probably validate/parse this in the view then just pass the collection exercise to this function directly. This would make it clearer what this function is doing

Reply:
I agree but I was just following the same pattern all the other endpoints have. Maybe have a refactor of all the controllers that implement the validation in the controller as part of another ticket if it is a big issue.